lib/api_constraints.rb

```diff
@@ -1,19 +1,14 @@
 class ApiConstraints
-  def initialize(version:)
-    @version = version
+  def initialize(options)
```

Comment:
This changes the method signature from always requiring a version, to the version being optional - I'm not sure if thats a good idea - it could instead be 

```
def initialize(version:, default: nil)
  @version = version
  @default = default
end
```

In general I think changing the API behaviour is a separate thing from this ticket should be in its own PR and reviewed independently

Reply:
I've already made this change @alexdesi (see master)

Reply:
We default from the value in the environment https://github.com/trade-tariff/trade-tariff-backend/pull/219